utils/frame/benchmarking-cli/src/shared/mod.rs

```diff
@@ -63,3 +65,49 @@ where
 	}
 	s
 }
+
+/// A Handlebars helper to add an underscore after every 3rd character,then format it,
+/// i.e. a separator for large numbers : 123000000 -> 123_000 * WEIGHT_PER_NANOS .
+#[derive(Clone, Copy)]
+pub struct UnderscoreFormatHelper;
+
+impl handlebars::HelperDef for UnderscoreFormatHelper {
+	fn call<'reg: 'rc, 'rc>(
+		&self,
+		h: &handlebars::Helper,
+		_: &handlebars::Handlebars,
+		_: &handlebars::Context,
+		_rc: &mut handlebars::RenderContext,
+		out: &mut dyn handlebars::Output,
+	) -> handlebars::HelperResult {
+		use handlebars::JsonRender;
+		let param = h.param(0).unwrap();
+		let underscore_format_param = underscore_format(param.value().render());
+		out.write(&underscore_format_param)?;
+		Ok(())
+	}
+}
+
+/// Add an underscore after every 3rd character, then format it, i.e. a separator for large numbers: 123000000 -> 123_000 * WEIGHT_PER_NANOS .
+fn underscore_format<Number>(i: Number) -> String
+where
+	Number: std::string::ToString,
+{
+	let mut s = String::new();
+	let i_str = i.to_string();
+	let f_u64 = i_str.parse::<f64>().unwrap();
+	let j_str = (f_u64 / constants::WEIGHT_PER_NANOS as f64 ).to_string();
+	let a = j_str.chars().rev().enumerate();
+	for (idx, val) in a {
+		if idx != 0 && idx % 3 == 0 {
+			if val=='.' {
+				
+			}else{
+				s.insert(0, '_');
+			}
+		}
+		s.insert(0, val);
+	}
+	s.push_str(" * constants::WEIGHT_PER_NANOS ");
```

Comment:
```suggestion
	s.push_str(" * WEIGHT_PER_NANOS");
```
This makes it a bit shorter. I think you already import the constant.

Reply:
yeah，that's better.